### ISSUE TITLE
Unable to get the valid object mesh with the provided sample bash command

### ISSUE THREAD

**Issue by jcliu0428** [xxlong0/SparseNeuS#24]
Dear authors,

Thank you for your great work. I tried to play with the bash command you provided:

**bash ./sample_bashs/dtu_scan118.sh**

The first 1k finetuning steps look good. But after finetuning 1k steps (2k or 3k steps), the object mesh turns to disappear and only background mesh is extracted.

Please take a good at the two figures for comparison (1st at 1k step, 2nd at 2k step): 

![image](https://user-images.githubusercontent.com/38600493/194932283-fda73175-c06f-46ec-a8a5-9098bb9b55e7.png)
![image](https://user-images.githubusercontent.com/38600493/194932387-2b7b7ddf-b96e-4bf0-a0c3-10a95539df5b.png)

Have you got any ideas to reproduce the correct mesh? Thank you for your time.

**Comment by flamehaze1115** [comment]
Hello. Did you run the most updated bash file?
`#!/usr/bin/env bash
python exp_runner_finetune.py \
--mode train --conf ./confs/finetune.conf --is_finetune \
--checkpoint_path ./weights/ckpt.pth \
--case_name scan118  --train_imgs_idx 0 1 2 --test_imgs_idx 0 1 2 --near 700 --far 1100 \
--visibility_beta 0.025 --visibility_gama 0.010 --visibility_weight_thred 0.7 0.6 0.5`

I have tested this command on RTX 2080Ti, The results are reasonable in the whole training.

**Comment by jcliu0428** [comment]
Yes. I have pulled your latest code and it still crashes after 2k steps. But I tested on a 3090 gpu, not sure whether this causes the issue. Thanks.

**Comment by xxlong0** [comment]
Can you share the experiment logs (the print out logs) with me? I can try to figure out the reason.

Best
Xiaoxiao
________________________________
发件人: jcliu0428 ***@***.***>
发送时间: Wednesday, October 12, 2022 1:56:05 AM
收件人: xxlong0/SparseNeuS ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [xxlong0/SparseNeuS] Unable to get the valid object mesh with the provided sample bash command (Issue #24)


Yes. I have pulled your latest code and it still crashes after 2k steps. But I tested on a 3090 gpu, not sure whether this causes the issue. Thanks.

―
